Makefile

```diff
@@ -1,6 +1,6 @@
-all: wasm-thumbnail-py/src/wasm_thumbnail/data/wasm_thumbnail.wasm
+all: wasm-thumbnail-py/wasm_thumbnail
 
 wasm-thumbnail-py/src/wasm_thumbnail/data/wasm_thumbnail.wasm: wasm-thumbnail/src/lib.rs
 	cd wasm-thumbnail && cargo build --release
-	cp wasm-thumbnail/target/wasm32-unknown-unknown/release/wasm_thumbnail.wasm wasm-thumbnail-py/src/wasm_thumbnail/data/
+	cp wasm-thumbnail/target/wasm32-unknown-unknown/release/wasm_thumbnail.wasm wasm-thumbnail-py/wasm_thumbnail/data
```

Comment:
I think something is slightly off with this, as running `make` will fail now

Reply:
If you create the dir like `wasm-thumbnail-py/wasm_thumbnail/data/` that will be fixed

Reply:
Perhaps the script should automatically generate that directory?

Reply:
Not yet, But I have to add some cmd to make it possible.